cmake/modules/AddSwift.cmake

```diff
@@ -1473,18 +1485,15 @@ function(add_swift_library name)
         endif()
 
         set(lipo_target "${name}-${SWIFT_SDK_${sdk}_LIB_SUBDIR}")
-        _add_swift_lipo_target(
-            ${sdk}
-            ${lipo_target}
-            ${UNIVERSAL_LIBRARY_NAME}
-            ${THIN_INPUT_TARGETS})
-
         if("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin" AND SWIFTLIB_SHARED)
```

Comment:
(in your quest to eliminate ${CMAKE_SYSTEM_NAME} STREQUAL "DARWIN" in favor of SDK checks)